sprout/scripts/config_validation/bgcf_schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "title": "All BGCF routes",
+  "type": "object",
+  "properties": {
+    "routes": {
+      "type": "array",
+      "items": {
+        "title": "Each BGCF route",
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "domain": {
+            "type": "string",
+            "description": "SIP trunk IP address or domain name"
```

Comment:
This [link](https://en.wikipedia.org/wiki/Domain_Name_System#Domain_name_syntax) explains what formats of domain name are valid: `